nav2_bringup/bringup/params/nav2_params.yaml

```diff
@@ -243,6 +243,21 @@ planner_server_rclcpp_node:
   ros__parameters:
     use_sim_time: True
 
+recoveries_server:
+  ros__parameters:
+    costmap_topic: local_costmap/costmap_raw
+    footprint_topic: local_costmap/published_footprint
+    cycle_frequency: 10.0
+    plugin_names: ["spin", "back_up", "wait"]
+    plugin_types: ["nav2_recoveries/Spin", "nav2_recoveries/BackUp", "nav2_recoveries/Wait"]
+    global_frame: odom
+    robot_base_frame: base_link
+    use_sim_time: true
```

Comment:
Do we also need to make changes to the following code to set this parameter?
https://github.com/ros-planning/navigation2/blob/9aafde139dffc3d69cfa6837145e0cca74f1ede7/nav2_bringup/bringup/launch/navigation_launch.py#L117

Reply:
Ah that's right, I missed this.